src/main/java/org/gridsuite/merge/orchestrator/server/CaseFetcherService.java

```diff
@@ -107,6 +105,19 @@ private String getSearchQuery(List<String> tsos, ZonedDateTime dateTime, String
         return Collections.emptyList();
     }
 
+    public List<FileInfos> getCases(List<UUID> caseUuids) {
+        List<FileInfos> cases = new ArrayList<>();
+        UriComponentsBuilder uriBuilder = UriComponentsBuilder.fromPath(DELIMITER + CASE_API_VERSION + "/cases/{caseUuid}");
```

Comment:
Don't use UriComponentsBuilder directly like this, it's unsafe. (in this case it works, but if there were weird characters in `(DELIMITER + CASE_API_VERSION + "/cases/{caseUuid}` or `queryParam("xiidm", false);` then it would break.

Instead, pass everything to resttemplate.exchange():
```java
String uriTemplate = DELIMITER + CASE_API_VERSION + "/cases/{caseUuid}?xiidm=false";
caseServerRest.exchange(uriTemplate, ..., caseUuid.toString())
```

Reply:
done